client/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces.ts

```diff
@@ -3,6 +3,7 @@ import { ReactNode } from 'react';
 export interface ItemsDataViewsType< T > {
 	items: T[] | undefined;
 	pagination: DataViewsPaginationInfo;
+	search?: boolean;
```

Comment:
I spent some time (a minute) to realize that the search field was about to enable/disable that feature and not about the text search. Maybe we should rename `search` to `enableSearchBox` and then `search={ data?.enableSearchBox ?? true }`